data-pipeline/events-router/src/main/config/events-router.properties

```diff
@@ -63,6 +63,7 @@ job.coordinator.replication.factor=1
 
 router.events.telemetry.route.topic=__env__.events.telemetry
 router.events.summary.route.topic=__env__.events.summary
+router.events.log.route.topic=__env__.events.log
```

Comment:
Have changes been made to create the Kafka topic using ansible?

Reply:
Updated ansible scripts